static/js/tools/scatter2/chart_loader.tsx

```diff
@@ -46,35 +46,49 @@ import { StatsVarNode } from "../timeline_util";
 interface Point {
   xVal: number;
   yVal: number;
-  xPop: number;
-  yPop: number;
   place: NamedPlace;
+  xDate: string;
+  yDate: string;
+  xSource: string;
+  ySource: string;
+  xPop?: number;
+  yPop?: number;
 }
 
-type Cache = Record<string, SourceSeries>;
+type Cache = {
```

Comment:
nit: add comment on what the key is (place or statVar)

Reply:
done